tests/Filesystem/FilesystemTest.php

```diff
@@ -324,4 +324,40 @@ public function testSharedGet()
         $this->assertTrue($result === 1);
         @unlink(__DIR__.'/file.txt');
     }
+
+    public function testRequireOnceRequiresFileProperly()
+    {
+        $filesystem = new Filesystem;
+        mkdir(__DIR__.'/foo');
+        file_put_contents(__DIR__.'/foo/foo.php', '<?php function random_function_xyz(){};');
+        $filesystem->requireOnce(__DIR__.'/foo/foo.php');
+        $this->assertTrue(function_exists('random_function_xyz'));
+        @unlink(__DIR__.'/foo/foo.php');
+        @rmdir(__DIR__.'/foo');
+    }
+
+    public function testCopyCopiesFileProperly()
+    {
+        $filesystem = new Filesystem;
+        $data = 'contents';
+        mkdir(__DIR__.'/foo');
+        file_put_contents(__DIR__.'/foo/foo.txt', $data);
+        $filesystem->copy(__DIR__.'/foo/foo.txt', __DIR__.'/foo/foo2.txt');
+        $this->assertTrue(file_exists(__DIR__.'/foo/foo2.txt'));
+        $this->assertEquals($data, file_get_contents(__DIR__.'/foo/foo2.txt'));
+        @unlink(__DIR__.'/foo/foo.txt');
+        @unlink(__DIR__.'/foo/foo2.txt');
+        @rmdir(__DIR__.'/foo');
+    }
+
+    public function testIsFileChecksFilesProperly()
+    {
+        $filesystem = new Filesystem;
+        mkdir(__DIR__.'/foo');
+        file_put_contents(__DIR__.'/foo/foo.txt', 'contents');
+        $this->assertTrue($filesystem->isFile(__DIR__.'/foo/foo.txt'));
+        $this->assertFalse($filesystem->isFile(__DIR__.'./foo'));
+        @unlink('/foo/foo.txt');
```

Comment:
This line should be:
```php
@unlink(__DIR__ . '/foo/foo.txt');
```

Currently, this leaves the `/foo/foo.txt` in the tests folder and breaks several tests if the test suite is run a second time.